src/clj/game/core/mark.clj

```diff
@@ -19,5 +20,5 @@
     (system-msg state :runner (str "identifies their mark to be " (central->name new-mark)))))
 
 (def identify-mark-ability
-  {:req (req (nil? (:mark @state)))
```

Comment:
I'm only okay with this if the CR says that "Identify Your Mark" always happens but game state changes only if there's no existing mark.

Reply:
I think that's what the text says for mark.
> When your turn begins, identify your mark. (If you don’t have a mark, a random central server becomes your mark for this turn.)

So identify should always happen, but the gamestate only changes the first time each turn.

Reply:
Ah yeah, god point. I forgot that's how they worded the reminder text. Cool!